twill-yarn/src/main/java/org/apache/twill/yarn/YarnTwillPreparer.java

```diff
@@ -183,6 +184,13 @@ public TwillPreparer withConfiguration(Map<String, String> config) {
   }
 
   @Override
+  public TwillPreparer withConfiguration(String runnableName, Map<String, String> config) {
```

Comment:
If the config contains only one key, say reserved memory, will the remaining keys default to the values configured for the entire app? Or will they default to the system default?